Project.toml

```diff
@@ -9,28 +9,30 @@ DataFrames = "a93c6f00-e57d-5684-b7b6-d8193f3e46c0"
 ExportAll = "ad2082ca-a69e-11e9-38fa-e96309a31fe4"
 FileIO = "5789e2e9-d7fb-5bc7-8068-2c6fae9b9549"
 Formatting = "59287772-0a20-5a39-b81b-1366585eb4c0"
-TheoryOfGames = "eb50afb4-6f20-4b37-9b66-473e668300bf"
+HiGHS = "87dc4568-4c63-4d18-b0c0-bb2238e4078b"
```

Comment:
Can you drop HiGHS?

Reply:
If i drop it I also have to change line 28 where It was used as:
# Read data from excel file
ECModel = ModelEC(input_file, EnergyCommunity.GroupCO(), HiGHS.Optimizer)